yamllint/rules/braces.py

```diff
@@ -103,23 +122,31 @@
 
 import yaml
 
+from yamllint.linter import LintProblem
 from yamllint.rules.common import spaces_after, spaces_before
 
 
 ID = 'braces'
 TYPE = 'token'
-CONF = {'min-spaces-inside': int,
+CONF = {'forbid': bool,
+        'min-spaces-inside': int,
         'max-spaces-inside': int,
         'min-spaces-inside-empty': int,
         'max-spaces-inside-empty': int}
-DEFAULT = {'min-spaces-inside': 0,
+DEFAULT = {'forbid': False,
+           'min-spaces-inside': 0,
            'max-spaces-inside': 0,
            'min-spaces-inside-empty': -1,
            'max-spaces-inside-empty': -1}
 
 
 def check(conf, token, prev, next, nextnext, context):
-    if (isinstance(token, yaml.FlowMappingStartToken) and
+    if conf['forbid'] and isinstance(token, yaml.FlowMappingStartToken):
+        yield LintProblem(token.start_mark.line + 1,
+                          token.end_mark.column + 1,
+                          'Flow mappings are forbidden to use')
```

Comment:
I propose
```suggestion
                          'forbidden flow mapping')
```
to be consistent with other messages (`forbidden implicit octal value`, `empty value in block mapping`, `comment not indented like content`, `found forbidden document end`...)

Reply:
Thanks for your suggestion! I applied it along with similar change in yamllint/rules/brackets.py, squashed and force-pushed.